acceptance/acceptance_test.go

```diff
@@ -109,6 +109,8 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 
 	if cloudEnv == "" {
 		server := testserver.New(t)
+		t.Cleanup(server.Close)
```

Comment:
Can we call `Close` from `testserver.New()` instead? Then it is impossible to use wrong.